src/libstd/sys/unix/process/process_unix.rs

```diff
@@ -237,7 +237,7 @@ impl Command {
     }
 
     #[cfg(not(any(target_os = "linux", target_os = "macos")))]
-    fn posix_spawn(&mut self, stdio: &ChildPipes, envp: Option<&CStringArray>)
+    fn posix_spawn(&mut self, _stdio: &ChildPipes, _envp: Option<&CStringArray>)
```

Comment:
Could you change this to `_`?

> Note that there is a subtle difference between using only _ and using a name that starts with an underscore. Something like _x still binds the value to the variable, whereas _ doesn’t bind at all. 

(https://doc.rust-lang.org/book/second-edition/ch18-03-pattern-syntax.html)

Reply:
Thanks, fixed.